en/news/_posts/2016-02-24-support--lan-of-ruby-200-and-21.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: news_post
+title: "Support plans for Ruby 2.0.0 and Ruby 2.1"
+author: "usa"
+translator:
+date: 2016-02-24 09:00:00 +0000
+lang: en
+---
+
+We announce about the future support plans for Ruby 2.0.0 and Ruby 2.1.
+
+## About Ruby 2.0.0
+
+As it has been announced from before, all support for Ruby 2.0.0 has ended today.
+Bug and security fixes from more recent Ruby versions will no longer be backported to 2.0.0, and no patch release of 2.0.0 will be released.
+
+We highly recommend that you upgrade to Ruby 2.3 or 2.2 as soon as possible.
+
+Please contact us via ruby-core ML if you'd like to continue maintaining the 2.0.0 branch for some justifiable reason you can't upgrade.
```

Comment:
I don't want to support someone who cannot upgrade :-P
We can accept some people who want to continue maintaining the 2.0.0 with understandable motivation.